dependencies/versions.go

```diff
@@ -21,6 +21,12 @@ import (
 	"github.com/pkg/errors"
 )
 
+type VersionUpdate struct {
```

Comment:
Maybe add a note that this is also the external schema for the JSON/YAML output.

Reply:
+1

Reply:
👍